### PR TITLE
edits for lab 1

### PR DESCRIPTION
**Pull request by iamjinakim**
fixed a few grammatical items and had some comments

### CODE REVIEW

File: workshop/Lab1/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ that has already been built and uploaded to DockerHub under the name
    This action will take a bit of time. To check the status of the running application,
```
Comment (iamjinakim) [inline workshop/Lab1/README.md]: Feels like "a bit of time" should be clarified unless this is something that devs will understand to mean a few minutes? hours?

File: workshop/Lab1/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 Learn how to deploy an application to a Kubernetes cluster hosted within
-the IBM Container Service.
+the IBM Cloud Kubernetes Service.
 
-# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
+## 1. Install prerequisite CLIs and provision a Kubernetes cluster
 
 If you haven't already:
 1. Install the IBM Cloud CLIs and login, as described in [Lab 0](../Lab0/README.md).
 2. Provision a cluster:
```
Comment (iamjinakim) [inline workshop/Lab1/README.md]: I assume that devs with a linux background wouldn't need further instructions on this?

**Comment by duglin** [comment]
@brahmaroutu please review

**Comment by stevemar** [comment]
Closing due to inactivity and conflicts

**Pull request closed without merging**